addons/rearm/functions/fnc_rearmEntireVehicleSuccessLocal.sqf

```diff
@@ -17,60 +17,42 @@
  */
 #include "script_component.hpp"
 
-params [["_truck", objNull, [objNull]], ["_vehicle", objNull, [objNull]], ["_turretPath", [], [[]]]];
-TRACE_3("rearmEntireVehicleSuccessLocal",_truck,_vehicle,_turretPath);
+params ["_truck", "_vehicle", "_turretPath"];
+TRACE_3("rearmEntireVehicleSuccessLocal", _truck, _vehicle, _turretPath);
```

Comment:
That's just how they work, for logging it's not a big problem, but it makes logs longer for no reason.